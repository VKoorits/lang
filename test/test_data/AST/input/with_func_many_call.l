print( "(i+1)^3 = ", pow(i+1, 3) )
