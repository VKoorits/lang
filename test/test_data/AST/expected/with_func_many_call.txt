CALL_0 NULL
CALL_0 :LIST_1 print NULL
LIST_1 :ELEM_2 CALL_3 NULL
ELEM_2 :(i+1)^3 =  NULL
CALL_3 :LIST_4 pow NULL
LIST_4 :ELEM_5 ELEM_6 NULL
ELEM_5 :i 1 + NULL
ELEM_6 :3 NULL
